internal/cmd/kafka/command_cluster_update.go

```diff
@@ -104,18 +178,17 @@ func (c *clusterCommand) validateResize(cku int32, currentCluster *cmkv2.CmkV2Cl
 	if currentCluster.Spec.GetAvailability() == highAvailability && cku <= 1 {
 		return 0, errors.New(errors.CKUMoreThanOneErrorMsg)
 	}
+
 	if cku == 0 {
 		return 0, errors.New(errors.CKUMoreThanZeroErrorMsg)
 	}
-	// Cluster can't be resized while it's provisioning or being expanded already.
-	// Name _can_ be changed during these times, though.
 	if err := isClusterResizeInProgress(currentCluster); err != nil {
 		return 0, err
 	}
+
 	// If shrink
-	if cku < currentCluster.GetSpec().Config.CmkV2Dedicated.Cku {
+	if cku < currentCluster.Spec.GetConfig().CmkV2Dedicated.GetCku() {
```

Comment:
Is there a reason why we removed the getter from spec here?

Reply:
It's a weird limitation of our SDK generator... hopefully the API team will fix this soon 😕

We've been using the getter on the last element of a chain of fields, because they're the most likely to be `nil` 😬